package.json

```diff
@@ -49,7 +49,7 @@
     "typescript": "^5.0.4"
   },
   "peerDepencencies": {
-    "eslint": ">=7.0.0 <9.0.0"
+    "eslint": ">=8.38.0 <9.0.0"
```

Comment:
No strong reason besides wanting to keep the ESLint internals more predictable (i.e., be recent) but not require that users be on the bleeding edge. I could probably open to 8.0 and temporarily modify the workflow to make sure everything works.

I chose 8.38 arbitrarily because that is what eslint-plugin-unicorn@47 requires.

Reply:
Following eslint-plugin-unicorn makes sense, and it's trivial for anyone using ESLint v8 to upgrade to at least ESLint 8.38, so I'm fine with it.